Sources/WebDriver/ElementLocator.swift

```diff
@@ -0,0 +1,52 @@
+/// A locator strategy to use when searching for an element.
+public struct ElementLocator: Codable, Hashable {
+    /// The locator strategy to use.
+    public var using: String
+    /// The search target.
+    public var value: String
+
+    public init(using: String, value: String) {
```

Comment:
For my understanding, could you please explain why the accessibilityId factory method was not added here, and was instead added in WinAppDriver\ElementLocator+accessibilityId.swift? Adding it here could have made the initializer private, forcing creation only by using factory methods.

Reply:
The "accessibility id" strategy is not part of the WebDriver spec but an extension of WinAppDriver. We want to allow usage of this library with arbitrary webdriver backends that might support other private strategies, so we need the public initializer.